problems/objects.js

```diff
@@ -187,15 +273,35 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = (str) => {
+  let obj = countOccurance(str)
+  delete obj[' ']
+  return obj
+};
 
+// console.log(countOccuranceNoSpaces("A good snake")) 
+// console.log(countOccuranceNoSpaces("HELLO world"))
 /**
  * Takes in an array and returns the most common element.
  * @param {Array} array - array of elements
  * @returns {(number|string)} Most common element
  */
 
-const mostCommonElement = () => {};
+const mostCommonElement = (array) => {// don't understand this one
+  let count = {}
+  for(let el of array){
+    count[el] = (count[el] || 0) + 1
+  }
+    let topValue = array[0];
+    let maxCount = 0;
+    for(let el of array){
```

Comment:
Instead of looping through the same array twice in a row, try doing everything in one loop.

Reply:
I made the correction for it